itest/list_on_test.go

```diff
@@ -269,14 +269,6 @@ var allTestCases = []*lntest.TestCase{
 		Name:     "psbt channel funding",
 		TestFunc: testPsbtChanFunding,
 	},
-	{
```

Comment:
nice combination! Initially I worried it would cause too many blocks being mined in a single test case, resulting a very unbalance tranche we've seen before, but turns out it's not bad!
```
=== RUN   TestLightningNetworkDaemon/tranche00/66-of-133/btcd/psbt_channel_funding
=== RUN   TestLightningNetworkDaemon/tranche00/66-of-133/btcd/psbt_channel_funding/anchors
    harness.go:339: finished test: psbt_channel_funding, start height=440, end height=473, mined blocks=33
=== RUN   TestLightningNetworkDaemon/tranche00/66-of-133/btcd/psbt_channel_funding/simple_taproot
    harness.go:339: finished test: psbt_channel_funding, start height=473, end height=506, mined blocks=33
=== NAME  TestLightningNetworkDaemon/tranche00/66-of-133/btcd/psbt_channel_funding
    harness.go:339: finished test: psbt_channel_funding, start height=440, end height=506, mined blocks=66
```